Resources/views/process/_process_list.html.twig

```diff
@@ -5,7 +5,11 @@
         <table class="table table-striped table-hover table-borderless">
             <tbody>
             {% for process in processes|slice(0, max_iterations) %}
+                {% if process.visitedByCaseWorker() %}
```

Comment:
Replace with one liner
<tr{% if not process.visitedByCaseWorker() %} class="not-visited"{% endif %}>

Reply:
Fixed in 96f59e1